fluster/decoders/iso_mpeg4_aac.py

```diff
@@ -25,16 +25,19 @@
 
 
 @register_decoder
-class FDKAACDecoder(Decoder):
-    '''FDK AAC reference decoder implementation'''
-    name = "FDK-AAC"
-    description = "FDK AAC reference decoder"
+class ISOAACDecoder(Decoder):
+    '''ISO MPEG4 AAC reference decoder implementation'''
+    name = "ISO-MPEG4-AAC"
+    description = "ISO MPEG4 AAC reference decoder"
     codec = Codec.AAC
-    binary = 'aac-dec'
+    binary = 'mp4audec_mc'
 
     def decode(self, input_filepath: str, output_filepath: str, output_format: OutputFormat, timeout: int,
                verbose: bool) -> str:
         '''Decodes input_filepath in output_filepath'''
+        # Addition of .pcm as extension is a must. If it is something else, e.g. ".out" it will output a ".wav" which
+        # which is undesirable.
+        output_filepath += ".pcm"
```

Comment:
Double which split into two different lines. Apart from this minor thingy, very nice work :+1:

Reply:
Thanks Pablo. I will correct with a fixup and then as discussed, rebase, squash and merge commit